lib/exchange/password.js

```diff
@@ -82,18 +83,18 @@ module.exports = function(options, issue) {
   }
 
   return function password(req, res, next) {
-    if (!req.body) { return next(new Error('OAuth2orize requires body parsing. Did you forget app.use(express.bodyParser())?')); }
-    
+    if (!req.body) { return next(new Error(g.f('{{OAuth2orize}} requires body parsing. Did you forget app.use(express.bodyParser())?'))); }
+
     // The 'user' property of `req` holds the authenticated user.  In the case
     // of the token endpoint, the property will contain the OAuth 2.0 client.
     var client = req[userProperty]
       , username = req.body.username
       , passwd = req.body.password
       , scope = req.body.scope;
-      
-    if (!username) { return next(new TokenError('Missing required parameter: username', 'invalid_request')); }
-    if (!passwd) { return next(new TokenError('Missing required parameter: password', 'invalid_request')); }
-    
+
+    if (!username) { return next(new TokenError(g.f('Missing required parameter: username', 'invalid_request'))); }
+    if (!passwd) { return next(new TokenError(g.f('Missing required parameter: password', 'invalid_request'))); }
+
```

Comment:
For both these lines, end bracket in the wrong place, should be in the first arg.